independent-publisher-connectors/snowflake/apiDefinition.swagger.json

```diff
@@ -0,0 +1,745 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Snowflake REST",
+    "description": "Snowflake connector based on Snowflake REST service",
+    "version": "1.0",
+	"contact": {

```

Comment:
@rekodus Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.